windows-icon-handler/refresh-icon-handler/refreshiconhandler_i.h

```diff
@@ -0,0 +1,240 @@
+
+

```

Comment:
Missing header

Reply:
This file is auto generated and shouldn't be part of our code base. I removed it